app/Service/Student/StudentBootcampDetailService.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service\Student;
+
+use App\Exceptions\StudentNotFoundException;
+use App\Exceptions\ResumeNotFoundException;
+use App\Exceptions\ResumeBootcampNotFoundException;
+use App\Models\Student;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+
+class StudentBootcampDetailService
+{
+    public function execute(string $studentId): array
+    {
+        try {
+            $student = Student::findOrFail($studentId);
+        } catch (ModelNotFoundException) {
+            throw new StudentNotFoundException($studentId);
+        }
+        $resume = $student->resume;
+        if (!$resume) {
+            throw new ResumeNotFoundException($studentId);
+        }
+        $bootcamps = $resume->bootcamps;
+        if (!$bootcamps) {
```

Comment:
Perhaps I'm mistaken, but I believe this block will never be executed because $bootcamps, even if it might be empty, will always exist.

I understand it has to do with the decision you've made about what should happen if there are no bootcamps, but in that case, this block might be unnecessary